File: 351001/Shimanchuk/RV1/src/main/java/by/bsuir/romamuhtasarov/impl/dto/NewsResponseTo.java
package by.bsuir.romamuhtasarov.impl.dto;

import lombok.Getter;
import lombok.Setter;

import java.sql.Date;

@Setter
@Getter
public class NewsResponseTo {
    private long id;
    private long writerId;
    private String title;
    private String content;
    private Date created;
    private Date modified;
}

File: 351002/Matsuyeu/Publisher/src/NewsPortal.Publisher.Application/Dtos/ResponseTo/NewsResponseTo.cs
namespace Publisher.src.NewsPortal.Publisher.Application.Dtos.ResponseTo
{
    // Response DTO - returned to client (no password)
    public class NewsResponseTo
    {
        public long Id { get; set; }
        public long CreatorId { get; set; }
        public string CreatorLogin { get; set; } = string.Empty;
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
        public List<string>? Marks { get; set; } = new List<string>();
    }
}
